code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -135,7 +150,7 @@
 			return 0
 
 	//If attempting to lable the storage item, silently fail to allow it
-	if(istype(W, /obj/item/hand_labeler) && user && user.a_intent != I_HELP)
+	if(istype(W, /obj/item/hand_labeler) || istype(W, /obj/item/forensics) && user && user.a_intent != I_HELP)
```

Comment:
Why? Super unintuitive interaction that should be reconsidered, but if this is something you really want you should (a) prompt the user with mechanical help and (b) have this be an item proc (`blocked_from_storage_on_help_intent` or whatever).

Reply:
I heard about issue when forensics items could be placed when you want to get swabs, or otherwise.